File: 01.CSharp-Programming-Basics/18.Book-Exercise/04.Simple-Conditions/Solution/SimpleConditions/AreaOfFigures/AreaOfFigures.cs
using System;

namespace AreaOfFigures
{
    class AreaOfFigures
    {
        static void Main(string[] args)
        {
            string typeFigure = Console.ReadLine().ToLower();

            double area = 0.0;

            switch (typeFigure)
            {
                case "square":
                    double a = double.Parse(Console.ReadLine());
                    area = a * a;
                    break;

                case "rectangle":
                    double b = double.Parse(Console.ReadLine());
                    double c = double.Parse(Console.ReadLine());
                    area = b * c;
                    break;

                case "circle":
                    double r = double.Parse(Console.ReadLine());
                    area = Math.PI * Math.Pow(r, 2);
                    break;

                case "triangle":
                    double e = double.Parse(Console.ReadLine());
                    double h = double.Parse(Console.ReadLine());
                    area = e * h / 2;
                    break;
            }

            Console.WriteLine($"{area}");
        }
    }
}

File: 02.JS-Programming-Basics/02.Conditional-Statements/Lab/AreaofFigures.js
function solve(typeFigure, firstArgument, secondArgument) {

    let area = 0;

    if (typeFigure == "square") {
        
        firstArgument = Number(firstArgument);
        area = firstArgument * firstArgument;

    } else if (typeFigure == "rectangle") {
        
        firstArgument = Number(firstArgument);
        secondArgument = Number(secondArgument);

        area = firstArgument * secondArgument;

    } else if (typeFigure == "circle") {
        
        firstArgument = Number(firstArgument);

        area = Math.PI * Math.pow(firstArgument, 2);

    } else {

        firstArgument = Number(firstArgument);
        secondArgument = Number(secondArgument);

        area = firstArgument * secondArgument / 2;
    }

    console.log(area.toFixed(3));
}

solve("circle", "6");
